ui/app/AppLayouts/Chat/popups/community/CreateChannelPopup.qml

```diff
@@ -109,7 +109,7 @@ StatusModal {
                 id: nameInput
                 label: qsTr("Channel name")
                 charLimit: popup.maxChannelNameLength
-                input.placeholderText: qsTr("Name the channel")
+                input.placeholderText: qsTr("# Name the channel")
```

Comment:
I remember we decided to not have this eventually because it may confuse the user if he needs to enter the hash. @John-44 @iurimatias maybe you both can confirm too? https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=2975%3A488617

Reply:
Will remove, it was in the design so I added it